homeassistant/components/camera/zoneminder.py

```diff
@@ -19,39 +18,14 @@
 DEPENDENCIES = ['zoneminder']
 DOMAIN = 'zoneminder'
 
-# From ZoneMinder's web/includes/config.php.in
-ZM_STATE_ALARM = "2"
-
-
-def _get_image_url(hass, monitor, mode):
-    zm_data = hass.data[DOMAIN]
-    query = urlencode({
-        'mode': mode,
-        'buffer': monitor['StreamReplayBuffer'],
-        'monitor': monitor['Id'],
-    })
-    url = '{zms_url}?{query}'.format(
-        zms_url=urljoin(zm_data['server_origin'], zm_data['path_zms']),
-        query=query,
-    )
-    _LOGGER.debug('Monitor %s %s URL (without auth): %s',
-                  monitor['Id'], mode, url)
-
-    if not zm_data['username']:
-        return url
-
-    url += '&user={:s}'.format(zm_data['username'])
-
-    if not zm_data['password']:
-        return url
-
-    return url + '&pass={:s}'.format(zm_data['password'])
-
 
 @asyncio.coroutine
 def async_setup_platform(hass, config, async_add_entities,
                          discovery_info=None):
     """Set up the ZoneMinder cameras."""
+    from zoneminder.zm import ZoneMinder
+    zm: ZoneMinder = hass.data[DOMAIN]
```

Comment:
I don't think this syntax is supported in Python 3.5.

Reply:
I thought type hinting was an official feature of Python 3.5?

Reply:
It is, but not this syntax. This is Python 3.6.

Reply:
Got it, with 3.5 it appears it can only be done for functions. I'll change this.